docs/tutorials/troubleshooting/get-console-session.md

```diff
@@ -1,3 +1,36 @@
 # Get a Console Session
+A console session can be helpful when VMs are not accessible through the normal network, for example, when the guest OS failed to configure the correct network settings during first boot. Follow the steps in this tutorial to get a console session to a VM using the `kubectl` command.
 
-// TODO ([github.com/vmware-tanzu/vm-operator#125](https://github.com/vmware-tanzu/vm-operator/issues/125))
+## Prerequisites
+Have edit or owner permissions on the namespace where the problematic VM is deployed. For more information, see [vSphere with Tanzu Identity and Access Management](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-concepts-planning/GUID-93B29112-4492-431F-958A-12323540C38D.html).
+
+## Procedure
+
+### 1. Access your namespace in the Kubernetes environment.
+See [Get and Use the Supervisor Context](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-services-workloads/GUID-63A1C273-DC75-420B-B7FD-47CB25A50A2C.html#GUID-63A1C273-DC75-420B-B7FD-47CB25A50A2C).
+
```

Comment:
Maybe add a stage `Confirm the permission required for creating webconsolerequests`?
```
$ kubectl auth can-i create webconsolerequests -n <NAMESPACE>
yes
```

Reply:
Good point! Added this command under the prerequisite section.